lib/cassanity/executors/cql_rb.rb

```diff
@@ -185,7 +184,7 @@ def call(args = {})
             end
 
             # Select the correct keyspace before executing the CQL query
-            if command != :keyspace_create && (keyspace_name = arguments[:keyspace_name])
+            if !command.to_s['keyspace'] && (keyspace_name = arguments[:keyspace_name])
```

Comment:
I believe symbols can match regex and maybe even starts with in newer versions of ruby. Away from my computer right now or I would try it.